cmd/diff-main.go

```diff
@@ -34,24 +34,19 @@ var (
 
 // Compute differences between two files or folders.
 var diffCmd = cli.Command{
-	Name:        "diff",
-	Usage:       "Show differences between two folders or buckets.",
-	Description: "Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. Objects of same name and size, but differ in contents are not noticed.",
-	Action:      mainDiff,
-	Before:      setGlobalsFromContext,
-	Flags:       append(diffFlags, globalFlags...),
+	Name:   "diff",
+	Usage:  "Show differences between two folders or buckets. Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. Objects of same name and size, but differ in contents are not noticed.",
```

Comment:
I think below `Usage` is better
```
List objects with size difference or missing between two folders or buckets
```

In `Description` you could have what algo it does like
```
Diff only lists missing objects or objects with size differences between given folders or buckets. It *DOES NOT* check for content difference of objects those having same name and size.
```

I can even think of removing `OUTPUT` section and put the info into `Description` like
```
Diff only lists missing objects or objects with size differences between given folders or buckets. It *DOES NOT* check for content difference of objects those having same name and size.

Differences are marked in output by:
    > - indicates object missing in destination.
    < - indicates object missing in the source.
    ! - indicates object size differed in source and destination.
```